src/base/bittorrent/torrentimpl.h

```diff
@@ -152,6 +152,9 @@ namespace BitTorrent
         bool hasMetadata() const override;
         bool hasMissingFiles() const override;
         bool hasError() const override;
+        bool hasWorkingTracker() const;
+        bool trackerNotWorking() const override;
+        bool m_hasWorkingTracker;
```

Comment:
Declare and initialize it in the private section instead of in the initializer list in the .cpp file: 
`bool m_hasWorkingTracker = true;` 
`m_xxxxx` are private in qBt and can be accessed by public getter/setter methods from other classes. You already have the getter: `trackerNotWorking()`. The setter may look kind of like the method `TorrentImpl::setName(const QString &name)`